main.js

```diff
@@ -112,9 +113,22 @@ app.on('ready', () => {
   uiWin.loadURL(`file://${__dirname}/ui/html/loading/index.html`);
 
   // Load app main page when qtumd is fully initialized
-  server.emitter.once('qtumd-started', () => {
+  server.emitter.once(ipcEvent.QTUMD_STARTED, () => {
     uiWin.loadURL(`http://${Config.HOSTNAME}:${Config.PORT}`);
   });
+
+  // Show error dialog if any startup errors
+  server.emitter.on(ipcEvent.STARTUP_ERROR, (err) => {
+    dialog.showMessageBox({
+      type: 'error',
+      buttons: ['Quit'],
```

Comment:
Question: how to localize string in graphql? should we move the electron logic to UI?

Reply:
I'm sure there are ways to localize it, but think it would have to be separate from UI logic since UI files are just static files and already built.

This dialog can't be moved to the UI layer since its the qtumd errors we are catching. And the localization lib in UI is specifically for react.